submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html>
+  <head>
+<meta charset "UTF-8">
+  <title>justify.com</title>
+  </head>
+
+  <body>
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+
+         <li><a href="#">New Releases</a></li>
```

Comment:
You are using `<li>` tag without `<ul>` or `<ol>` tag. This is wrong

Reply:
Reviewed again. I have effected the changes. Please check.